### ISSUE TITLE
Backward compatibility for Ruby 2.5.x + 2.6.x

### ISSUE THREAD

**Issue by robbyrussell** [testdouble/cypress-rails#100]
Bonjour!

We were attempting to introduce cypress-rails to an existing Ruby on Rails application that is running on Ruby 2.5.x at the moment. 

### Stacktrace

With the most recent version(s) of cypress-rails, it fails on the following error: 

(snipped the useful bits)

```ruby
Traceback (most recent call last):
	5: from /Users/robbyrussell/.rbenv/versions/2.5.8/lib/ruby/gems/2.5.0/gems/cypress-rails-0.5.1/exe/cypress-rails:13:in `<main>'
	4: from /Users/robbyrussell/.rbenv/versions/2.5.8/lib/ruby/gems/2.5.0/gems/cypress-rails-0.5.1/lib/cypress-rails/open.rb:11:in `call'
	3: from /Users/robbyrussell/.rbenv/versions/2.5.8/lib/ruby/gems/2.5.0/gems/cypress-rails-0.5.1/lib/cypress-rails/launches_cypress.rb:20:in `call'
	2: from /Users/robbyrussell/.rbenv/versions/2.5.8/lib/ruby/gems/2.5.0/gems/cypress-rails-0.5.1/lib/cypress-rails/manages_transactions.rb:10:in `begin_transaction'
	1: from /Users/robbyrussell/.rbenv/versions/2.5.8/lib/ruby/gems/2.5.0/gems/cypress-rails-0.5.1/lib/cypress-rails/manages_transactions.rb:59:in `gather_connections'
/Users/robbyrussell/.rbenv/versions/2.5.8/lib/ruby/gems/2.5.0/gems/cypress-rails-0.5.1/lib/cypress-rails/manages_transactions.rb:74:in `setup_shared_connection_pool': undefined method `bind_call' for #<UnboundMethod:0x00007fe9006c3c70> (NoMethodError)
```

### Affected Versions

* cypress-rails `v0.5.2`
* cypress-rails `v0.5.1`

### Likely Cause

Given that `bind_call` was introduced in Ruby 2.7.x ([link](https://github.com/ruby/ruby/commit/83c6a1ef454c51ad1c0ca58e8a95fd67a033f710)), and cypress-rails took advantage of this in 38e935753d08886802f87adaf285bad44c181573 ([jump to change](https://github.com/testdouble/cypress-rails/commit/38e935753d08886802f87adaf285bad44c181573#diff-fcdb99c436a6f17005b7cd99fc9aff85b3d3bc061f9fdf016ccbc1cae774da0a)), it'll cause problems for apps running on older versions of Ruby.

Do you anticipate wanting to continue supporting 2.5.x and 2.6.x?

**Comment by robbyrussell** [comment]
In the meantime, we can begin to take advantage of this delightful gem by running v0.5.0.

    gem 'cypress-rails', '0.5.0'

Until next time!

**Comment by searls** [comment]
Thanks for pointing this out. Fixed in 0.5.3
